wp-modules/app/js/src/utils/toKebabCase.ts

```diff
@@ -1,5 +1,5 @@
-/** Converts a string to a slug. */
-export default function convertToSlug( toConvert = '' ) {
+/** Converts a string to a kebab-case. */
+export default function toKebabCase( toConvert = '' ) {
```

Comment:
The name `converToSlug()` was confusing, because it wasn't to convert to a pattern slug. 

Just kebab-case.